services/http/api/lib/models/measurement.ts

```diff
@@ -1,18 +1,38 @@
 import { ModelRelatedNodesI, NeogmaInstance, NeogmaModel } from "neogma";
-import { UserModel, UserInstance } from "./user";
-import { DateInstance, DateModel } from "./date";
+// import { DateInstance, DateModel } from "./date";
+import { DayInstance, DayModel, HourInstance, HourModel, MonthInstance, MonthModel, TimestampInstance, TimestampModel, YearInstance, YearModel } from "./time";
+import { SourceInstance, SourceModel } from "./source";
+import { UserInstance, UserModel } from "./user";
+
+export type MeasurementTypeProperties = {
+    name: string,
+}
+
+export type MeasurementTypeRelatedNodesI = {
+    Measurement: ModelRelatedNodesI<MeasurementModel, MeasurementInstance>,
+    Source: ModelRelatedNodesI<SourceModel, SourceInstance>;
+};
+
+export type MeasurementTypeInstance = NeogmaInstance<MeasurementTypeProperties, MeasurementTypeRelatedNodesI>;
+
+export type MeasurementTypeModel = NeogmaModel<MeasurementTypeProperties, MeasurementTypeRelatedNodesI>;
+
+// ----------------------------------------------------------------------------------------------------------
```

Comment:
I usually find it cleaner if we use code comment blocks or just comments for separation rather than these types of separators. You can have two here, one at the top of the `MeasurementType` section and another at the top of the `Measurement` section here. If we want to use separators like this, we should keep them consistent across files. I saw in another file we are using `=`.

Reply:
Make sense, kind of like an explicit section labelling? 

Added!